src/overlays/actors/ovl_En_In/z_en_in.c

```diff
@@ -1477,15 +1609,15 @@ void func_808F6334(EnIn* this, PlayState* play) {
     Player* player = GET_PLAYER(play);
     s32 talkState = Message_GetState(&play->msgCtx);
 
-    this->unk4C4 += this->unk4C0 != 0.0f ? 40.0f : -40.0f;
+    this->unk4C4 += (this->unk4C0 != 0.0f) ? 40.0f : -40.0f;
     this->unk4C4 = CLAMP(this->unk4C4, 0.0f, 80.0f);
 
     Matrix_Translate(this->unk4C4, 0.0f, 0.0f, MTXMODE_APPLY);
     if ((&this->actor == player->talkActor) &&
-        !((play->msgCtx.currentTextId >= 0xFF) && (play->msgCtx.currentTextId <= 0x200)) &&
-        (talkState == TEXT_STATE_3) && (this->prevTalkState == TEXT_STATE_3)) {
+        ((play->msgCtx.currentTextId < 255) || (play->msgCtx.currentTextId > 0x200)) && (talkState == TEXT_STATE_3) &&
```

Comment:
```suggestion
        ((play->msgCtx.currentTextId < 0xFF) || (play->msgCtx.currentTextId > 0x200)) && (talkState == TEXT_STATE_3) &&
```
This should probably stay as hex

Reply:
Oops, don' know why I did that.

I feel like `(msgCtx->currentTextId >= 0x100) && (msgCtx->currentTextId <= 0x200)` should be a macro given how many times that shows up along with `(play->msgCtx.currentTextId < 0xFF) || (play->msgCtx.currentTextId > 0x200)`, but they aren't quite not's of each other because it would be `!((play->msgCtx.currentTextId >= 0xFF) && (play->msgCtx.currentTextId <= 0x200))` which is 1 off